lib/pbench/server/api/resources/users_api.py

```diff
@@ -456,33 +464,25 @@ def delete(self, username):
                     }
         """
         try:
-            user, verified = self.auth.verify_user(username)
-        except Exception:
-            self.logger.exception("Exception occurred during the getUser {}", username)
-            abort(500, message="INTERNAL ERROR")
-
-        # TODO: Check if the user has the right privileges
-        if not verified and not user.is_admin():
-            self.logger.warning(
-                "User {} is not authorized to delete user {}.", user.username, username,
-            )
-            abort(403, message="Not authorized to delete user")
+            target_user = self.operation_permitted(target_username)
+            if not target_user:
+                self.logger.warning(
+                    "User {} is not authorized to delete user {}.",
+                    self.auth.token_auth.current_user().username,
+                    target_username,
+                )
+                abort(HTTPStatus.FORBIDDEN, message="Not authorized to delete user")
+            if target_user.is_admin():
+                self.logger.warning(
+                    "Admin user is not allwed to self delete via API call. Username: {}",
```

Comment:
"allowed" ... and maybe just "the Admin user cannot be deleted via API call". But further... we don't really need the username here until we have a real distinction between those "is admin" vs "has admin" cases we'd discussed earlier; and I suspect there's no point in trying to over-generalize at this time since there'll be a lot of changes through this code when we add roles.

Reply:
Since this is the log, I think we _do_ want the username.  (We'll want as much context as we can get if we need to investigate this retroactively.)